app/importers/concerns/importable.rb

```diff
@@ -90,6 +90,10 @@ def normalize_industry(industry)
     end
   end
 
+  def get_mapper_terms_from_array(array)
+    array.map { |i| normalize_industry(i) }.compact.flatten.uniq
+  end
```

Comment:
I'm a huge fan of this. A better solution would be to add this functionality to the gem, i.e. you could pass an array of terms to IndustryMappingClient#map_industry and it would do the rest (including managing caching). It's possible to look up several terms at once via the IM API, and taking advantage of that would lead to the most efficient solution.

However, rather than ask you to make such changes, I've deferred them to [another story](https://www.pivotaltracker.com/story/show/110477482). It would be too much of a detour to ask you to do them as part of this PR.